samples/bluetooth/throughput/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -0,0 +1,6 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_NCS_SAMPLES_DEFAULTS=n
```

Comment:
Why it's disabled here only for this board?

Reply:
For having better shell and logger cooperation the same on nRF5340

Reply:
Please add explanation in the file about this.

Reply:
Done